src/sarif_converter.ts

```diff
@@ -81,8 +81,9 @@ const getTools = (analysisResults: IAnalysisResult, suggestions: ISarifSuggestio
       },
       properties: {
         tags: [language, ...suggestion.tags, ...suggestion.categories],
+        exampleCommitFixes: suggestion.exampleCommitFixes,
         precision: 'very-high',
-      } as { tags: string[]; precision: string; cwe?: string[] },
+      } as { tags: string[]; exampleCommitFixes?: ExampleCommitFix[]; precision: string; cwe?: string[] },
```

Comment:
Question: don't we need `exampleCommitDescriptions` as well? (worth asking @Spoor2709 )

Reply:
I think this will be useful information to have 😃

Reply:
Alright then, adding it 😄

Reply:
Done